libs/dev-dock/frontend/src/dev_dock.tsx

```diff
@@ -83,22 +82,11 @@ function ElectionControl(): JSX.Element | null {
 
   const selectedElection = getElectionQuery.data;
 
-  async function onSelectElection(
+  function onSelectElection(
     event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>
   ) {
     const path = event.target.value;
-    if (path === 'Pick from file...') {
-      const dialogResult = await assertDefined(window.kiosk).showOpenDialog({
-        properties: ['openFile'],
-      });
-      if (dialogResult.canceled) return;
-      const selectedPath = dialogResult.filePaths[0];
-      if (selectedPath) {
-        setElectionMutation.mutate({ path: selectedPath });
-      }
-    } else {
-      setElectionMutation.mutate({ path });
-    }
```

Comment:
This feature is currently quite important in dev dock if you want to use mock smart cards with any election other than the fixtures. I don't think we can get rid of it without an alternative.

Reply:
Added this feature back using the same approach, so we still need `kiosk.showOpenDialog`.